drawing_agent/rag/vectors.py

```
@@ -1,52 +1,52 @@
 
     def _load_or_create(self):
         if os.path.exists(self.index_path) and os.path.exists(self.metadata_path):
             self.index = faiss.read_index(self.index_path)
-            with open(self.metadata_path, "rb") as f:
-                self.metadata = pickle.load(f)
+            with open(self.metadata_path, "r", encoding="utf-8") as f:
+                self.metadata = json.load(f)
         else:
-            # Создаём плоский индекс inner product (после нормализации даёт косинусное сходство)
-            self.index = faiss.IndexFlatIP(384)  # размерность эмбеддинга 384 для 'all-MiniLM-L6-v2'
+            # Мы НЕ создаем индекс сразу, так как не знаем размерность
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `Н` (CYRILLIC CAPITAL LETTER EN). Did you mean `H` (LATIN CAPITAL LETTER H)?

(RUF003)